### DISCUSSION TITLE
Battery life drain while using Zed

### DISCUSSION THREAD

**Discussion by ayhon** [zed-industries/zed#6591] [Feature Requests]
I was wondering if battery drain was significantly greater when using Zed on a laptop. Since the GUI is rendered using the GPU and this is usually the most powerhungry part of a laptop, I would assume that battery life would be reduced while running Zed. Is this actually true? If not, how is this addressed?

**Comment by dipankardas011** [comment]
Yes I myself have seen this and not sure why does a editor need GPU?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/community-links
